### PR TITLE
Deprecate Sign only method in 2.0 Spec

### PR DESCRIPTION
**Pull request by Funkatronics**

### CODE REVIEW

File: spec/spec.md [hunk 1]
```diff
@@ -123,16 +123,21 @@ Feature identifiers are used to identify features that are supported by a wallet
 These features are mandatory and must be implemented by wallet endpoints. Dapp endpoints can assume that these features are always available without an explicit call to [`get_capabilities`](#get_capabilities).  
 
 - [`solana:signMessages`](#sign_messages)
-- [`solana:signTransactions`](#sign_transactions)
+- [`solana:signAndSendTransaction`](#sign_and_send_transactions), an optional Wallet RPC method.
```
Comment (jordansexton) [inline spec/spec.md:126]: > These features are mandatory

vs.

> an optional Wallet RPC method.

File: spec/spec.md [hunk 2]
```diff
@@ -573,67 +578,6 @@ This method can be used to enumerate the capabilities and limits of a wallet end
...
-    - `valid`: an array of booleans with the same length as payloads indicating which are valid
-
-- `ERROR_NOT_SIGNED` if the wallet endpoint declined to sign these transactions for any reason
-- `ERROR_TOO_MANY_PAYLOADS` if the wallet endpoint is unable to sign all transactions due to exceeding implementation limits. These limits may be available via [`get_capabilities`](#get_capabilities).
-
-##### Description
-
-The wallet endpoint should attempt to simulate the transactions provided by data and present them to the user for approval (if applicable). If approved (or if it does not require approval), the wallet endpoint should sign the transactions with the private keys for the requested authorized account addresses, and return the signed transactions to the dapp endpoint.
-
 #### sign_and_send_transactions
```
Comment (jordansexton) [inline spec/spec.md:581]: Spec for `sign_transactions` removed but spec for `sign_and_send_transactions` not added?

File: spec/spec.md (hunk 1 above)
Comment (Funkatronics) [reply spec/spec.md:126]: haha yes will fix

File: spec/spec.md (hunk 2 above)
Comment (Funkatronics) [reply spec/spec.md:581]: sign_and_send_transactions was already in the spec

Comment (Funkatronics) [reply spec/spec.md:581]: the sign and send spec might need to be updated, however

**Review by sdlaver — APPROVED** [review]

**Pull request merged**